packages/device-utils/CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+# 1.0.1
+
+-   fix(device-utils): add tsx as dev dependency (75e2cf82af)
+-   npm-prerelease: @trezor/device-utils 1.0.1-beta.1 (65fb5950b5)
+-   chore(device-utils): prepare setup for npm release (e4c32edb37)
+-   chore: more refactorign to remove connect from components (3736dec0a5)
+-   chore: move DeviceModelInternal to device-utils (0bcf666672)
+-   chore(monorepo): remove dependency on connect in components and device-utils (465f40a072)
```

Comment:
these records make no sense. it probably takes some default max number of records because it wasn't able to find previsou changelog commit..